cmds/webboot/webboot.go

```diff
@@ -142,16 +142,16 @@ func (d *DownloadOption) exec(uiEvents <-chan ui.Event, network bool, cacheDir s
 	if err != nil {
 		return nil, err
 	}
-
 	var link string
 	if entry.Label() == customLabel {
 		link, err = menu.PromptTextInput("Enter URL:", validURL, uiEvents)
 		if err != nil {
 			return nil, err
 		}
 	} else {
-		distro := supportedDistros[entry.Label()]
-		link = distro.url
+		mirrorNameForTestPurposes := ""
+		_ = mirrorNameForTestPurposes
+		link, mirrorNameForTestPurposes, err = mirrorMenu(entry, uiEvents, link)
```

Comment:
Combine these lines by using := on the third line.

Reply:
Please add a check for the `err` value returned..

```
if err != nil {
    return nil, err
}
```

Reply:
Also, do not forget to combine these lines with `:=`.